src/core/ColorUtil.h

```diff
@@ -24,8 +25,16 @@ void rgbtohsv(float r, float g, float b, float& h, float& s, float& v);
 
 std::optional<Color4f> parse_hex_color(const std::string& hex);
 
+std::optional<Color4f> parse_web_color(const std::string& col);
```

Comment:
parse_web_color(),  parse_hex_color(), and rgbtohsv() are implementation details and can be removed from the header.